OTAcceleratorCore/OTOneToOneCommunicator.m

```diff
@@ -180,23 +182,31 @@ - (void)notifiyAllWithSignal:(OTCommunicationSignal)signal error:(NSError *)erro
 
 #pragma mark - OTSessionDelegate
 -(void)sessionDidConnect:(OTSession*)session {
-    
-    [[LoggingWrapper sharedInstance].logger setSessionId:session.sessionId
+    LoggingWrapper *loggingWrapper = [LoggingWrapper sharedInstance];
+    [loggingWrapper.logger setSessionId:session.sessionId
                                             connectionId:session.connection.connectionId
                                                partnerId:@([self.session.apiKey integerValue])];
+    [loggingWrapper.logger logEventAction:KLogActionConnect
+                                                 variation:KLogVariationSuccess
+                                                completion:nil];
     
     [[LoggingWrapper sharedInstance].logger logEventAction:KLogActionEndCommunication
                                                  variation:KLogVariationSuccess
                                                 completion:nil];
     
     if (!self.publisher) {
-        
+        [loggingWrapper.logger logEventAction:KLogActionStartPublishing
+                                                     variation:KLogVariationAttempt
+                                                    completion:nil];
         if (!self.screenSharingView) {
             OTPublisherSettings *setting = [[OTPublisherSettings alloc] init];
             setting.name = self.name;
             self.publisher = [[OTPublisher alloc] initWithDelegate:self settings:setting];
         }
         else {
+            [loggingWrapper.logger logEventAction:KLogActionStartScreensharing
+                                                         variation:KLogVariationAttempt
+                                                        completion:nil];
```

Comment:
Under screen sharing, it will log both. Should we put KLogActionStopPublishing to the else branch?